docs/src/main/asciidoc/logging.adoc

```diff
@@ -101,115 +152,133 @@ class SimpleBean {
 WARN:: A non-critical service error or problem that may not require immediate correction.
-INFO:: Service lifecycle events or important related very-low-frequency information.
-DEBUG:: Messages that convey extra information regarding lifecycle or non-request-bound events which may be helpful for debugging.
+INFO:: Service lifecycle events or important related very low-frequency information.
+DEBUG:: Messages that convey extra information regarding lifecycle or non-request-bound events, useful for debugging.
 TRACE:: Messages that convey extra per-request debugging information that may be very high frequency.
-ALL:: Special level for all messages including custom levels.
+ALL:: A special level to use in configuration to turn on logging for all messages, including custom levels.
 
-In addition, the following levels may be configured for applications and libraries using link:https://docs.oracle.com/javase/8/docs/api/java/util/logging/Level.html[`java.util.logging`]:
+You can also configure the following levels for applications and libraries that use link:https://docs.oracle.com/javase/8/docs/api/java/util/logging/Level.html[`java.util.logging`]:
 
 [horizontal]
 SEVERE:: Same as **ERROR**.
 WARNING:: Same as **WARN**.
 CONFIG:: Service configuration information.
 FINE:: Same as **DEBUG**.
 FINER:: Same as **TRACE**.
-FINEST:: Event more debugging information than `TRACE`, maybe with even higher frequency.
+FINEST:: Increased debug output compared to `TRACE`, which might have a higher frequency.
+
+.The mapping between the levels
+[options="header",cols="^33%,^33%,^33%"]
+|===
+|Numerical level value |Standard level name |Equivalent `java.util.logging` (JUL) level name
+
+|1100 |FATAL | Not applicable
+
+|1000 |ERROR |SEVERE
+
+|900 |WARN |WARNING
+
+|800 |INFO |INFO
+
+|700 | Not applicable |CONFIG
+
+|500 |DEBUG |FINE
+
+|400 |TRACE |FINER
+
+|300 | Not applicable |FINEST
+|===
+
 
 == Runtime configuration
 
-Run time logging is configured in the `application.properties` file,
-for example, to set the default log level to `INFO` logging and include Hibernate `DEBUG` logs:
+Runtime logging is configured in the `application.properties` file.
+
+Because JBoss Logging is built-in to Quarkus, link:https://quarkus.io/developer-joy/[unified configuration] is provided for all <<supported-logging-apis,supported logging APIs>>.
 
+.An example of how you can set the default log level to `INFO` logging and include Hibernate `DEBUG` logs:
 [source, properties]
 ----
 quarkus.log.level=INFO
 quarkus.log.category."org.hibernate".level=DEBUG
 ----
 
-Setting a log level below `DEBUG` requires the minimum log level to be adjusted,
-either globally via the `quarkus.log.min-level` property or per-category as shown in the example above,
-as well as adjusting the log level itself.
+When you set the log level to below `DEBUG`, you must adjust the minimum log level to one of the following options:
+
+Directly:: By adjusting the log level directly.
```

Comment:
Not sure what this means. This doesn't follow from the previous text. Something like this should be enough:

```asciidoc
When you set the log level to below `DEBUG`, you must also adjust the minimum log level.
This setting is either global, using the `quarkus.log.min-level` configuration property, or per category:

[source, properties]
----
quarkus.log.category."org.hibernate".min-level=TRACE
----

It sets a floor level for which Quarkus needs to generate supporting code.
The minimum log level must be set at build time, so that Quarkus can open the door to optimization opportunities by eliding logging for levels that can never be used.
```

Reply:
Deluxe, applying :)